actions/setup_e2e_tests.sh

```diff
@@ -59,11 +56,6 @@ if [[ -n "$RHTEST" ]]; then
         rm -rf bats-core
     fi
 
```

Comment:
Just removing these lines will break pretty much all of the ChatOps end-to-end tests, since they rely on the `bats` command being available. Instead of installing `bats` from its GitHub repo, we should install BATS from its RPM package with YUM. You might want to double check the repos for the exact name, but I believe it's just `bats`.